public/index.html

```diff
@@ -12,12 +12,12 @@
     <header class="bg-pink-700 text-white py-4">
         <nav class="container mx-auto flex justify-between items-center animate__animated animate__zoomIn">
             <div class="text-2xl font-bold animate__animated animate__fadeIn">
-                <a class="hover:text-black" href="https://docs.google.com/document/d/1DH8-hbCTK8OqZzKmxjY30_ghpfESRpVt/edit?usp=drivesdk&ouid=110576272185341512116&rtpof=true&sd=true">Blogs</a>
+                <a class="hover:text-black hover:transition-colors hover:duration-300" href="https://docs.google.com/document/d/1DH8-hbCTK8OqZzKmxjY30_ghpfESRpVt/edit?usp=drivesdk&ouid=110576272185341512116&rtpof=true&sd=true">Blogs</a>
```

Comment:
Don't prefix `hover:` when adding `duration`. `duration` property should the at the top level. Otherwise it will only work when we are hovering not when we move away from the link.